CMakeLists.txt

```diff
@@ -839,16 +843,16 @@ if(CONFIG_ARM AND CONFIG_USERSPACE)
 
   set(PRIV_STACKS_OUTPUT_OBJ_PATH ${CMAKE_CURRENT_BINARY_DIR}/CMakeFiles/priv_stacks_output_lib.dir/${PRIV_STACKS_OUTPUT_OBJ})
 
+  set(objCpyCmd "")
+  bintools_objcopy(
+    RESULT_CMD_LIST objCpyCmd
+    SECTION_RENAME  ".bss=.priv_stacks.noinit;.data=.priv_stacks.data;.text=.priv_stacks.text;.rodata=.priv_stacks.rodata"
```

Comment:
The readability here degraded a lot because we no longer have alignment. Is it somehow possible to read these from a list? E.g. something like:

set(priv_stacks_section_renamings
.bss=.priv_stacks.noinit
.data=.priv_stacks.data
)

Reply:
Yes, you are correct, it is not easy to see the individual sections being renamed. 
I will try out the list you suggest, and update the PR after.

Reply:
The renamed segments has now been placed into lists, as you suggested.
I have done that for all calls to objcopy, having more than one segment.